CRM/CiviAwards/Setup/CreateApplicantReviewActivityType.php

```diff
@@ -45,4 +46,28 @@ private function addApplicantReviewActivityType() {
     ]);
   }
 
+  /**
+   * Sets activity status 'Scheduled' and 'Completed' for applicant
+   * review activity type if not already set.
+   */
+  private function setApplicantReviewActivityTypeStatuses() {
+    $statuses = civicrm_api3('OptionValue', 'get', [
+      'name' => ['IN' => ['Scheduled', 'Completed']],
+      'option_group_id' => 'activity_status',
+    ])['values'];
+
+    foreach ($statuses as $status) {
+      $groupings = explode(',', $status['grouping']);
+      $index = array_search('Applicant Review', $groupings);
+      if ($index !== FALSE){
+        continue;
+      }
+
+      civicrm_api3('OptionValue', 'create', [
+        'id' => $status['id'],
+        'grouping' => "{$status['grouping']},Applicant Review",
```

Comment:
What is {$status['grouping']} is empty or NULL? In that case we won't need the comma before applicant review